src/component/handlers/edit/editOnCopy.js

```diff
@@ -31,9 +31,29 @@ function editOnCopy(editor: DraftEditor, e: SyntheticClipboardEvent): void {
     return;
   }
 
-  editor.setClipboard(getFragmentFromSelection(editor._latestEditorState));
-  if (editor.props.onCopy) {
-    editor.props.onCopy(e);
+  const fragment = getFragmentFromSelection(editor._latestEditorState);
+  editor.setClipboard(fragment);
+  if (editor.props.convertBlockMapToClipboard) {
+    const clipboardDataToPush = editor.props.convertBlockMapToClipboard(fragment);
```

Comment:
We should move this into its own file and include it in editOnCopy and editOnCut.